jekyll/_layouts/classic-docs.html

```diff
@@ -95,7 +95,5 @@ <h5 class="toc-heading">On this page</h5>
   {% include_cached js-assets.html %}
 </div><!-- .outter -->
 {% include popup_cta.html %}
-
-{% include_cached no-script.html %}
```

Comment:
are we sure we need to remove this? Is it not going to break our future getting started experiment?